src/UnitTests/UnitTests.csproj

```diff
@@ -65,6 +65,10 @@
     <LangVersion>5</LangVersion>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="nunit.framework, Version=3.0.5813.39035, Culture=neutral, PublicKeyToken=2638cd05610744eb, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\packages\NUnit\lib\nunit.framework.dll</HintPath>
+    </Reference>
     <Reference Include="System" />
```

Comment:
I don't think this reference should be changed. Maybe my recent changes to use paket dependency groups (NuGet) have confused some tooling?